tests/DatabasePreparationTest.php

```diff
@@ -86,29 +85,24 @@
     });
 });
 
-// Helpers
-/**
-     * Run the database seeds.
-     *
-     * @return void
-     */
-function run()
+class User extends Authenticable
 {
-    DB::table('users')->insert([
-        'name' => 'Seeded User',
-        'email' => 'seeded@user',
-        'password' => bcrypt('password'),
-    ]);
+    protected $guarded = [];
 }
 
-function __construct(Tenant $tenant)
+class CreateSuperuser
```

Comment:
Was it `Superuser` in the original branch (`master`) or `SuperUser`?

Reply:
It is Superuser. 
https://github.com/archtechx/tenancy/blob/master/tests/DatabasePreparationTest.php#L124